src/components/bridge/ethereum/L1Bridge.vue

```diff
@@ -134,7 +134,8 @@
         <span class="color--white"> {{ $t(errMsg) }}</span>
       </div>
 
-      <div v-if="fromChainName === EthBridgeNetworkName.AstarZk" class="row--box-error">
+      <!-- <div v-if="fromChainName === EthBridgeNetworkName.AstarZk" class="row--box-error"> -->
+      <div v-if="false" class="row--box-error">
```

Comment:
@gtg7784 How about creating a variable like `const isWarningHighTraffic = false` at this file?

Reply:
Oh yeah, that looks more better